data/IL/incentives.json

```diff
@@ -0,0 +1,769 @@
+    ],
+    "short_description": {
+      "en": "$40 rebate for ENERGY STAR certified Electric Clothes Dryer."
+    },
+    "end_date": 2023
+  },
+  {
+    "id": "IL-2",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 rebate for ENERGY STAR certified Heat Pump Clothes Dryer."
+    },
+    "end_date": 2023
+  },
+  {
+    "id": "IL-3",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_stove",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 100
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$100 rebate for Induction Cooktop."
+    },
+    "end_date": 2023
+  },
+  {
+    "id": "IL-4",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "pos_rebate",
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 off ENERGY STAR certified Smart Thermostats."
+    },
+    "end_date": 2023
+  },
+  {
+    "id": "IL-5",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "weatherization",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $2 off on Weatherization Products such as caulking, weatherstripping, insulation, and door sweeps/seals."
+    },
+    "end_date": 2023
+  },
+  {
+    "id": "IL-6",
+    "authority_type": "utility",
+    "authority": "il-corn-belt-energy-corporation",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "il_cornBeltEnergyCorporation_rebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 750
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750 rebate for replacing existing geothermal with new geothermal system with an electric backup."
+    }
+  },
+  {
+    "id": "IL-7",
+    "authority_type": "utility",
+    "authority": "il-corn-belt-energy-corporation",
+    "payment_methods": [
+      "unknown"
```

Comment:
Outstanding question if this method should be unknown. See Slack thread: https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708466841224109

Reply:
Ticket created here: https://app.asana.com/0/1206629812394927/1206629812396080/f

I'm not sure who would be expecting to manage these tickets though? Maybe move back into an incentive board after the swarm is over? @veekas

Reply:
Just flagging that this will eventually block IL from being moved to a launched state due to https://github.com/rewiringamerica/api.rewiringamerica.org/blob/1ae3463106091755ba23ada194bda49e7b5833d4/test/data/schemas.test.ts#L211.

That's fine for now, but that's something I put in place to ensure we don't start serving these before we've figure out better answers.

Reply:
I tried my luck by reaching out to the number on the [website](https://www.cornbeltenergy.com/geocents) 800-879-0339 to get more information. The representative took my contact info, but we will see how fruitful this turns out to be 🤞🏼.